2022-03-10 16:34:44,506 INFO    MainThread:53975 [wandb_setup.py:_flush():75] Loading settings from /share/etud/e2008984/.config/wandb/settings
2022-03-10 16:34:44,506 INFO    MainThread:53975 [wandb_setup.py:_flush():75] Loading settings from /share/mastoc/projects/erasmus/dl_historical_images/utils/wandb/settings
2022-03-10 16:34:44,506 INFO    MainThread:53975 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-03-10 16:34:44,506 INFO    MainThread:53975 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-03-10 16:34:44,506 INFO    MainThread:53975 [wandb_init.py:_log_setup():405] Logging user logs to /share/mastoc/projects/erasmus/dl_historical_images/utils/wandb/run-20220310_163444-3971kt3l/logs/debug.log
2022-03-10 16:34:44,507 INFO    MainThread:53975 [wandb_init.py:_log_setup():406] Logging internal logs to /share/mastoc/projects/erasmus/dl_historical_images/utils/wandb/run-20220310_163444-3971kt3l/logs/debug-internal.log
2022-03-10 16:34:44,507 INFO    MainThread:53975 [wandb_init.py:init():439] calling init triggers
2022-03-10 16:34:44,507 INFO    MainThread:53975 [wandb_init.py:init():442] wandb.init called with sweep_config: {}
config: {}
2022-03-10 16:34:44,507 INFO    MainThread:53975 [wandb_init.py:init():458] re-initializing run, found existing run on stack: 243dnya1
2022-03-10 16:34:44,512 INFO    MainThread:53975 [wandb_run.py:finish():1539] finishing run vibferreira/master_thesis/243dnya1
2022-03-10 16:34:44,513 INFO    MainThread:53975 [jupyter.py:save_history():429] not saving jupyter history
2022-03-10 16:34:44,513 INFO    MainThread:53975 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-03-10 16:34:44,513 INFO    MainThread:53975 [wandb_init.py:_jupyter_teardown():337] cleaning up jupyter logic
2022-03-10 16:34:44,513 INFO    MainThread:53975 [wandb_run.py:_atexit_cleanup():1702] got exitcode: 0
2022-03-10 16:34:44,517 INFO    MainThread:53975 [wandb_run.py:_restore():1674] restore
2022-03-10 16:34:44,957 INFO    MainThread:53975 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 709
  total_bytes: 709
}

2022-03-10 16:34:45,255 INFO    MainThread:53975 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 709
  total_bytes: 709
}

2022-03-10 16:34:45,802 INFO    MainThread:53975 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 2
}
pusher_stats {
  uploaded_bytes: 709
  total_bytes: 993
}

2022-03-10 16:34:45,904 INFO    MainThread:53975 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 709
  total_bytes: 4223
}

2022-03-10 16:34:46,008 INFO    MainThread:53975 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 709
  total_bytes: 4223
}

2022-03-10 16:34:46,111 INFO    MainThread:53975 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 4131
  total_bytes: 4223
}

2022-03-10 16:34:46,213 INFO    MainThread:53975 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 4131
  total_bytes: 4223
}

2022-03-10 16:34:46,315 INFO    MainThread:53975 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 4223
  total_bytes: 4223
}

2022-03-10 16:34:46,418 INFO    MainThread:53975 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 4223
  total_bytes: 4223
}

2022-03-10 16:34:46,520 INFO    MainThread:53975 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 4223
  total_bytes: 4223
}

2022-03-10 16:34:46,623 INFO    MainThread:53975 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 4223
  total_bytes: 4223
}

2022-03-10 16:34:46,726 INFO    MainThread:53975 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 4223
  total_bytes: 4223
}

2022-03-10 16:34:47,219 INFO    MainThread:53975 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 4223
  total_bytes: 4223
}

2022-03-10 16:34:47,657 INFO    MainThread:53975 [wandb_run.py:_on_finish():1831] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 4223
  total_bytes: 4223
}
local_info {
}

2022-03-10 16:34:48,769 INFO    MainThread:53975 [wandb_run.py:_footer_history_summary_info():2865] rendering history
2022-03-10 16:34:48,771 INFO    MainThread:53975 [wandb_run.py:_footer_history_summary_info():2894] rendering summary
2022-03-10 16:34:48,778 INFO    MainThread:53975 [wandb_run.py:_footer_sync_info():2822] logging synced files
2022-03-10 16:34:48,785 INFO    MainThread:53975 [wandb_init.py:init():492] starting backend
2022-03-10 16:34:48,785 INFO    MainThread:53975 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-03-10 16:34:48,793 INFO    MainThread:53975 [backend.py:ensure_launched():219] starting backend process...
2022-03-10 16:34:48,817 INFO    MainThread:53975 [backend.py:ensure_launched():224] started backend process with pid: 102724
2022-03-10 16:34:48,819 INFO    MainThread:53975 [wandb_init.py:init():501] backend started and connected
2022-03-10 16:34:48,831 INFO    MainThread:53975 [wandb_run.py:_label_probe_notebook():947] probe notebook
2022-03-10 16:34:48,832 INFO    MainThread:53975 [wandb_run.py:_label_probe_notebook():957] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-03-10 16:34:48,832 INFO    MainThread:53975 [wandb_init.py:init():565] updated telemetry
2022-03-10 16:34:48,833 INFO    MainThread:53975 [wandb_init.py:init():596] communicating run to backend with 30 second timeout
2022-03-10 16:34:51,162 INFO    MainThread:53975 [wandb_run.py:_on_init():1759] communicating current version
2022-03-10 16:34:51,234 INFO    MainThread:53975 [wandb_run.py:_on_init():1763] got version response 
2022-03-10 16:34:51,234 INFO    MainThread:53975 [wandb_init.py:init():625] starting run threads in backend
2022-03-10 16:34:53,008 INFO    MainThread:53975 [wandb_run.py:_console_start():1733] atexit reg
2022-03-10 16:34:53,009 INFO    MainThread:53975 [wandb_run.py:_redirect():1606] redirect: SettingsConsole.WRAP
2022-03-10 16:34:53,010 INFO    MainThread:53975 [wandb_run.py:_redirect():1643] Wrapping output streams.
2022-03-10 16:34:53,011 INFO    MainThread:53975 [wandb_run.py:_redirect():1667] Redirects installed.
2022-03-10 16:34:53,012 INFO    MainThread:53975 [wandb_init.py:init():664] run started, returning control to user process
2022-03-10 16:35:26,642 INFO    MainThread:53975 [wandb_setup.py:_flush():75] Loading settings from /share/etud/e2008984/.config/wandb/settings
2022-03-10 16:35:26,643 INFO    MainThread:53975 [wandb_setup.py:_flush():75] Loading settings from /share/mastoc/projects/erasmus/dl_historical_images/utils/wandb/settings
2022-03-10 16:35:26,643 INFO    MainThread:53975 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-03-10 16:35:26,643 INFO    MainThread:53975 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-03-10 16:35:26,643 INFO    MainThread:53975 [wandb_init.py:_log_setup():405] Logging user logs to /share/mastoc/projects/erasmus/dl_historical_images/utils/wandb/run-20220310_163526-3q451psj/logs/debug.log
2022-03-10 16:35:26,643 INFO    MainThread:53975 [wandb_init.py:_log_setup():406] Logging internal logs to /share/mastoc/projects/erasmus/dl_historical_images/utils/wandb/run-20220310_163526-3q451psj/logs/debug-internal.log
2022-03-10 16:35:26,643 INFO    MainThread:53975 [wandb_init.py:_jupyter_setup():355] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7fc7c8208e50>
2022-03-10 16:35:26,644 INFO    MainThread:53975 [wandb_init.py:init():439] calling init triggers
2022-03-10 16:35:26,644 INFO    MainThread:53975 [wandb_init.py:init():442] wandb.init called with sweep_config: {}
config: {}
2022-03-10 16:35:26,644 INFO    MainThread:53975 [wandb_init.py:init():458] re-initializing run, found existing run on stack: 3971kt3l
2022-03-10 16:35:26,646 INFO    MainThread:53975 [wandb_run.py:finish():1539] finishing run vibferreira/master_thesis/3971kt3l
2022-03-10 16:35:26,646 INFO    MainThread:53975 [wandb_run.py:_atexit_cleanup():1702] got exitcode: 0
2022-03-10 16:35:26,648 INFO    MainThread:53975 [wandb_run.py:_restore():1674] restore
2022-03-10 16:35:27,168 INFO    MainThread:53975 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 709
  total_bytes: 709
}

2022-03-10 16:35:27,287 INFO    MainThread:53975 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 709
  total_bytes: 709
}

2022-03-10 16:35:27,494 INFO    MainThread:53975 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 709
  total_bytes: 709
}

2022-03-10 16:35:28,249 INFO    MainThread:53975 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 709
  total_bytes: 4127
}

2022-03-10 16:35:28,352 INFO    MainThread:53975 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 709
  total_bytes: 4127
}

2022-03-10 16:35:28,455 INFO    MainThread:53975 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 709
  total_bytes: 4127
}

2022-03-10 16:35:28,557 INFO    MainThread:53975 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1570
  total_bytes: 4127
}

2022-03-10 16:35:28,660 INFO    MainThread:53975 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1570
  total_bytes: 4127
}

2022-03-10 16:35:28,762 INFO    MainThread:53975 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 4127
  total_bytes: 4127
}

2022-03-10 16:35:28,865 INFO    MainThread:53975 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 4127
  total_bytes: 4127
}

2022-03-10 16:35:28,968 INFO    MainThread:53975 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 4127
  total_bytes: 4127
}

2022-03-10 16:35:29,070 INFO    MainThread:53975 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 4127
  total_bytes: 4127
}

2022-03-10 16:35:29,173 INFO    MainThread:53975 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 4127
  total_bytes: 4127
}

2022-03-10 16:35:29,625 INFO    MainThread:53975 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 4127
  total_bytes: 4127
}

2022-03-10 16:35:30,073 INFO    MainThread:53975 [wandb_run.py:_on_finish():1831] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 4127
  total_bytes: 4127
}
local_info {
}

2022-03-10 16:35:31,246 INFO    MainThread:53975 [wandb_run.py:_footer_history_summary_info():2865] rendering history
2022-03-10 16:35:31,247 INFO    MainThread:53975 [wandb_run.py:_footer_history_summary_info():2894] rendering summary
2022-03-10 16:35:31,250 INFO    MainThread:53975 [wandb_run.py:_footer_sync_info():2822] logging synced files
